Core/ConfigManager.cs

```diff
@@ -4,11 +4,9 @@
 
 namespace Core
 {
-    public class ConfigManager
-    {
-        public static List<Credentials> GetCredentials()
-        {            
-            return JsonConvert.DeserializeObject<List<Credentials>>(File.ReadAllText("credentials.json"));
-        }
-    }
+	public class ConfigManager
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931949)